src/js/components/List/List.js

```diff
@@ -400,6 +459,8 @@ const List = React.forwardRef(
                 } else if (Array.isArray(adjustedBackground)) {
                   adjustedBackground =
                     adjustedBackground[index % adjustedBackground.length];
+                } else if (isPinned) {
+                  adjustedBackground = theme.list.item.pinned.color;
```

Comment:
Since we're using this theme value for the background of the item, I think the theme path should be `theme.list.item.pinned.background`